RxSwift/Traits/PrimitiveSequence.swift

```diff
@@ -148,13 +148,13 @@ extension PrimitiveSequence {
     }
 
     /**
-     Repeats the source observable sequence the specified number of times in case of an error or until it successfully terminates.
+     Runs the source observable sequence the specified number of times in case of an error or until it successfully terminates.
 
      If you encounter an error and want it to retry once, then you must use `retry(2)`
```

Comment:
@freak4pc If your last comment is true, then this line of documentation is very confusing.

> It is not 100% true because
> ```
> thing
>  .retry(1)
> ```
> will actually have 2 tries (initial and a single retry) , so the terminology of "at most the specified number" might be confusing. (even if correct)

Reply:
So maybe I'm wrong? That could very well be true haha :) Let me confirm.

Reply:
Whoops, I totally am. Let me remove my last comments to not confuse anyone reading later.

Reply:
@freak4pc does my confusion make sense now? :) Do you have any thoughts on improved wording?

Reply:
I was sure I replied to this a few days ago. Ha! Too much work 😅